deploy-commands.js.js

```diff
@@ -1,29 +1,37 @@
-const { REST, Routes } = require("discord.js");
-const { clientId, guildId, token } = require("./config.json");
-const fs = require("fs");
+// deploy-commands.js
+import "dotenv/config";
+import { REST, Routes } from "discord.js";
+import fs from "fs";
+import path from "path";
+import { fileURLToPath } from "url";
+
```

Comment:
Can use our env helper for validation:
```suggestion
import { required } from "./utils/envHelpers.js";

```